app/src/main/java/me/thuanc177/kotsune/viewmodel/MangaViewModel.kt

```
@@ -0,0 +1,218 @@
+                    var coverArtFileName: String? = null
+
+                    for (j in 0 until relationships.length()) {
+                        val rel = relationships.getJSONObject(j)
+                        if (rel.getString("type") == "cover_art") {
+                            coverArtFileName = rel.getJSONObject("attributes").getString("fileName")
+                            break
+                        }
+                    }
+
+                    coverArtFileName
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)